### PR TITLE
Geometry IA: Formatting the answers

### PR DESCRIPTION
**Pull request by adityatandon007**
## Description of new Instant Answer, or changes
This PR does minor changes to the answers displayed by this IA.

## Related Issues and Discussions
This PR does small changes in its `yaml` file (formatting the results to be displayed)

## People to notify
@moollaza @pjhampton 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/geometry

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by adityatandon007** [comment]
@moollaza Please review and if it looks good merge this immediately so that I can add my changes that we have talked in #3163

**Comment by pjhampton** [comment]
> if it looks good merge this immediately

I have been burned in the past merging things that look good but haven't been tested properly. I think regardless of the fix we need follow a stricter review process (more so than the one we already have). 

Regardless, we'll try to expedite this today 👍  Thanks for all your hard work @adityatandon007

**Comment by mintsoft** [comment]
@pjhampton If only we could make it so community leaders could deploy to beta. Or autodeploying to beta

**Comment by pjhampton** [comment]
Hey @mintsoft 

> autodeploying to beta

No dice, unfortunately. Anything perl backed needs to be human skimmed before it goes into beta to make sure there was nothing malicious added. Maybe if none of the perl was touched... but I'm not familar with the beta environment. It's closed source and tends to lag a couple of weeks behind the live site.

As for comm leaders deploying, I don't see why not. They can already deploy Fatheads AFAIK. I'll bring it up with @moollaza later 👍

**Comment by adityatandon007** [comment]
@pjhampton thanks for looking into this PR. The reason why I said 

> if it looks good merge this immediately

is because yesterday @moollaza confirmed that this IA has been converted into goodie and suggested if I can made the changes that my earlier PR was supposed to do when this wasn't a goodie (you remember when I asked you to put the label: `on hold`) so I thought of porting that PR but then I saw some irregularities in the way results are displaying so just thought lets made a separate PR for this and then port my shapes into other PR (as @moollaza always say `keep one issue to one PR`).

P.S. I would be happy waiting till everyone is okay testing what this PR does(basically, does `no change`) and then move to porting my previous PR into this goodie.

**Comment by moollaza** [comment]
> Or autodeploying to beta

@mintsoft I'm working on that for all non-Perl PRs. Believe me I want it as badly as everyone else!

**Comment by moollaza** [comment]
Deploying to Beta 👍

File: share/goodie/geometry/objectInfo.yml [hunk 1]
```diff
@@ -22,8 +22,8 @@ cube:
   diagonal: a√3
 cuboid:
   volume: abc
-  surface: 2(ab + ac + bc)
-  diagonal: "√(a<sup>2</sup> + b<sup>2</sup> + c<sup>2</sup>)"
+  surface: 2(ab+ac+bc)
```
Comment (moollaza) [inline share/goodie/geometry/objectInfo.yml]: I think I actually prefer with the extra spacing as it's easier to read, this is OK tho

**Comment by adityatandon007** [comment]
@moollaza thanks for merging this PR, now I can add extra shapes like cylinder and cone geometry into this IA and then we can have a vote whether it looks good with spaces or without spaces. It looks good to me more this way(without spaces).

**Pull request merged**